### PR TITLE
Adding Discoin

### PR DESCRIPTION
**Pull request by cherrries**
# Pull Request Template

## Description

Please include a short summary of the changes in this PR.

## Type of change

- [x] Metadata related change
- [ ] Other

## Checklist:

- [x] For metadata related changes, this PR code passes the GitHub Actions metadata validation

## Metadata PRs

Please note it may take up to 4 hours for merged changes to take effect on the metadata server.

### CODE REVIEW

**Comment by cleanerm5** [comment]
Hi @cherrries 

the properties name, decimals, descriptions and url are signed twice with the same keys. Please remove the redundant one from the file. Otherwise looks ok from my end.

File: mappings/5612bee388219c1b76fd527ed0fa5aa1d28652838bcab4ee4ee63197446973636f696e.json [hunk 1]
```diff
@@ -0,0 +1,67 @@
+{
+    "name": {
+        "signatures": [
+            {
+                "publicKey": "3de04d6dc537dc6b90be40cfdb998e608ed6e791aa89bc077c3e5e235715658d",
+                "signature": "b63fc11a5572197d9be843e6300113853e11e8d1282a49a6693ca2c9407a9042c6f269de3647ad363330db0ca119a250a19df7c1621d088bb3af6dbc8419e605"
```
Comment (v-e-k) [inline mappings/5612bee388219c1b76fd527ed0fa5aa1d28652838bcab4ee4ee63197446973636f696e.json:6]: removed redundant signatures

**Comment by cleanerm5** [comment]
@cherrries looks good. As soon as the pipeline no longer hangs I'll do the merge (if it passes the verification).

**Review by cleanerm5 — APPROVED** [review]

File: mappings/5612bee388219c1b76fd527ed0fa5aa1d28652838bcab4ee4ee63197446973636f696e.json [hunk 2]
```diff
@@ -0,0 +1,67 @@
...
...R7xLjp8ssiy5KaKv9GoyJl/gqqHNyUcTQNPy1B4DxR5n3BEjRkpDhaXhF9jT26jIkDlldK/L7FJDcopaRqY1AFq7hlCMRCVo5bqjyhTuqqxxDOVbZcvtr2irZvD1TQiXzuA94agBKEaQ7nNBVJNbJpXtIggmxahiJSmCVVLjvdQizKUnNbYxKAMoYGxXZgVJxE+i3T36yDSgCY7k64I6V5RkgUiWCM5OhlF+eKf/tS8SmRh1ewhftbWgp8ci0a1QBScfU9xynm21ZIUt4JjlVbJnqSYh7srn0VxrwPL3kN+KCn2m500yUt+7I3lf1XYWWmV4rrNy1DlJKYIWFui261Jjp3wIHIb23woI2CNKYRBYacm5MpP3wrC1kmkR6ncUdplmzj+awOd7LebxHiavN54kqJpmzE2v8KWGGpE59ZbErqX9hUx+KJ4QTovXZAtyReV1aaAc2v02qZhTUcVjaZNadN0eKp/GaB5T6U1SqsU16lgFO28RX4dAuwYoRJE5UNT+T2cf1VD8QAJgwr2U9tdfb1sw1YRaiQtXDs299TI6oesvuVf/9R6syJykQAbJ4+2DLI7ld1jlK2QtNnYZmvT5k0WfrLzHpoOiN1jZO0OvO329jso7SGSoHi2a6wirt0oselnKvzQFkcmSSadEFDc5V/VNv1LozxX/OlKgXgLnUiX6O6FKDti2NE+He0kByiVQ1c4NJ/p6RLf9vfG/7nDo+54t072kzHbQkDsWoD3IC13jrCLHVtZeDszbzfQ7nU66YLOlndVkuDmY+DmRLi9Iwr/LenB/HaZ9v8rYPf1/Oc/4e2/NTdh1/8O8L8D/JP09l+6AMT+A8r/2sUzmj6q/+YAuw2TrbNgp7z+9d5ll//fE+xpHAv/L3b/H/ySK2+7bExOUf2TAL3Ni29TEu7RbDYJt3+KZOyD790EgP9FsrklRhbIf91eGW4XwP8XGxZwUmgyd3MAAAAxdEVYdGNvbW1lbnQAT3B0aW1pemVkIHdpdGggaHR0cHM6Ly9lemdpZi5jb20vb3B0aW1pemUYjL0nAAAAJXRFWHRkYXRlOmNyZWF0ZQAyMDIyLTA2LTE2VDIxOjQ2OjAwKzAwOjAw6hgYKwAAACV0RVh0ZGF0ZTptb2RpZnkAMjAyMi0wNi0xNlQyMTo0NTo1OSswMDowMK0KUCMAAAAASUVORK5CYII="
```
Comment (cherrries) [inline mappings/5612bee388219c1b76fd527ed0fa5aa1d28652838bcab4ee4ee63197446973636f696e.json:40]: oculus -- here

**Pull request merged**